modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/item/selector/select_kb_articles.jsp

```diff
@@ -0,0 +1,179 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/item/selector/init.jsp" %>
+
+<%
+KBArticleItemSelectorViewDisplayContext kbArticleItemSelectorViewDisplayContext = (KBArticleItemSelectorViewDisplayContext)request.getAttribute(KBArticleItemSelectorViewDisplayContext.class.getName());
+%>
+
+<clay:management-toolbar
+	managementToolbarDisplayContext="<%= new KBArticleItemSelectorViewManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, kbArticleItemSelectorViewDisplayContext) %>"
+/>
+
+<clay:container-fluid
+	cssClass="item-selector lfr-item-viewer"
+	id='<%= liferayPortletResponse.getNamespace() + "articlesContainer" %>'
+>
+	<liferay-site-navigation:breadcrumb
+		breadcrumbEntries="<%= kbArticleItemSelectorViewDisplayContext.getPortletBreadcrumbEntries() %>"
+	/>
+
+	<liferay-ui:search-container
+		emptyResultsMessage="no-knowledge-base-article-was-found"
+		id="articles"
+		searchContainer="<%= kbArticleItemSelectorViewDisplayContext.getSearchContainer() %>"
+	>
+		<liferay-ui:search-container-row
+			className="Object"
+			modelVar="object"
+		>
+
+			<%
+			KBArticle curKBArticle = null;
+			KBFolder curKBFolder = null;
+
+			Object result = row.getObject();
+
+			if (result instanceof KBFolder) {
+				curKBFolder = (KBFolder)result;
+			}
+			else {
+				curKBArticle = kbArticleItemSelectorViewDisplayContext.getLatestArticle((KBArticle)result);
+			}
+			%>
+
+			<c:choose>
+				<c:when test="<%= curKBArticle != null %>">
+					<liferay-ui:search-container-column-text>
+						<liferay-ui:user-portrait
+							userId="<%= curKBArticle.getUserId() %>"
+						/>
+					</liferay-ui:search-container-column-text>
+
+					<liferay-ui:search-container-column-text
+						colspan="<%= 2 %>"
+					>
+						<div class="item-preview selector-button" data-returnType="<%= kbArticleItemSelectorViewDisplayContext.getKBArticleDataReturnType() %>" data-value="<%= HtmlUtil.escapeAttribute(kbArticleItemSelectorViewDisplayContext.getKBArticleDataValue(curKBArticle)) %>">
```

Comment:
I guess now the `.item-preview` class is not necessary, as the selector here is `.selector-button` class. Could you test if this class can me revomed?

Reply:
I added the `item-preview` class because it gives the "hand" icon on the mouse to be the same behavior than other item selectors.

Reply:
Thats ok then, thanks!